common/v2/features/ScreenLock/ScreenLockProvider.tsx

```diff
@@ -47,6 +48,18 @@ class ScreenLockProvider extends Component<
     startLockCountdown: (lockingOnDemand: boolean) => this.startLockCountdown(lockingOnDemand)
   };
 
+  public decrypt = (data: string, key: number[]) => {
+    const aes = new ModeOfOperation.ctr(key);
+    const decryptedBytes = aes.decrypt(utils.hex.toBytes(data));
+    return utils.utf8.fromBytes(decryptedBytes);
+  };
+
+  public encrypt = (data: string, key: Uint8Array) => {
+    const aes = new ModeOfOperation.ctr(key);
+    const encryptedBytes = aes.encrypt(utils.utf8.toBytes(data));
+    return utils.hex.fromBytes(encryptedBytes);
+  };
+
```

Comment:
Definitely utility functions.
Extract from component and test please.
Add a comment, name or scope (could stay in the same in file) so we know that's it's only used for our password handling and not chain related.